data/ccd-update-data.json

```diff
@@ -0,0 +1,3 @@
+{
+    "D8ScreenHasMarriageBroken": "NO"
```

Comment:
why we need separate file can we add this to above ccd-basic-data.json ?

Reply:
This is the update case json, whilst the basic data is for submission?